doc/book/replication/repl_leader_elect.rst

```diff
@@ -103,9 +103,20 @@ Once the leader is elected, it considers itself in the leader position until rec
 This can lead to the split situation if the other nodes elect a new leader upon losing the connectivity to the previous one.
 
 The issue is resolved in Tarantool version :doc:`2.10.0 </release/2.10.0>` by introducing the leader *fencing* mode.
-The mode can be switched on and off by the :ref:`election_fencing_enabled <repl_leader_elect_config>` configuration parameter.
-When the fencing is on, the leader resigns its leadership if it has less than the :ref:`replication_synchro_quorum <repl_leader_elect_config>`
-of alive connections to the cluster nodes. The resigning leader receives the status of a follower in the current election term and becomes read-only.
+The mode can be switched by the :ref:`election_fencing_mode <repl_leader_elect_config>` configuration parameter.
+When the fencing is set to ``soft`` or ``strict``, the leader resigns its leadership if it has less than the
+:ref:`replication_synchro_quorum <repl_leader_elect_config>` of alive connections to the cluster nodes.
+The resigning leader receives the status of a follower in the current election term and becomes read-only.
+Leader *fencing* can be turned off by setting :ref:`election_fencing_mode <repl_leader_elect_config>` configuration parameter to ``off``.
```

Comment:
```suggestion
Leader *fencing* can be turned off by setting the :ref:`election_fencing_mode <repl_leader_elect_config>` configuration parameter to ``off``.
```
And here the article "the" is attached to the word "parameter", so it is required.